### ISSUE TITLE
Redirection Error

### ISSUE THREAD

**Issue by Silvertech-CS** [johngodley/redirection#824]
```
Plugin: 3.2
WordPress: 4.9.4 (single)
PHP: 5.6.31-4+wpengine10
Browser: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.186 Safari/537.36
JavaScript: https://www.oneideaaway.com/wp-content/plugins/redirection/redirection.js
REST API: https://www.oneideaaway.com/wp-json/

Error: Failed to fetch
```

Hi John, I have this on a staging site on wpengine and it works just fine, but on the live site it is not fetching. Here are the things I have done:

- I pressed the Magic Fix button, but still no fetching
- I looked at the https://www.oneideaaway.com/wp-json/ address to see if that was working.. It looks like it, lots of data, so I assume it is working. I followed your information on the Wordpress REST API page. We do not have any security plugins on the site. We do have cloudflare which leads me to...
- I followed your directions for cloudflare, cleared cache, set up page rules.. nada nada nada
- Last but not least. I do have a lot of plugins activated, BUT I checked them against the staging site where the redirection plugin is working just fine. The same plugins are activated on both sites. 

So, not sure what may be happening. Please help! 

Steve

**Comment by Silvertech-CS** [comment]
Just for more information... the Plugin Status reads... 

Wordpress REST API - REST API is being redirected. This indicates it has been disabled or you have a trailing slash redirect. (301 - https://www.oneideaaway.com/wp-json/redirection/v1/)

Redirection routes - REST API is not working so routes not checked

The rest are all good. :)

**Comment by Silvertech-CS** [comment]
I have tried everything I can on Cloudflare, but I have a feeling the problem lies there and the fact it is versioning the redirection.js file, but not positive. I copied the live site down to a staging site and it worked absolutely fine. It is just the live site so that has me pointed at Cloudflare. I was able to solve this problem by some of the settings in Cloudflare. I put it in development mode.
